### DISCUSSION TITLE
Does your FPCA have Varimax rotation?

### DISCUSSION THREAD

**Discussion by hovinh** [GAA-UAM/scikit-fda#382] [Q&A] [answered]
Hi scikit-fda team,

According to the book "Applied Functional Data Analysis: Methods and Case Studies", applying Varimax rotation on FPCA can help attain more informative components. I wonder if this is implemented already in your FPCA method? 

Thank you,
Vinh

**Comment by vnmabus** [comment, marked as answer]
No, we do not have that implemented.

**Reply by hovinh** [reply]
Thank you. Do you have it in your development plan? Can I contribute it? I have read the mentioned book, but the implementation details are not clear. However, I'm willing to learn if I can have some guidance from you or the team.

**Reply by vnmabus** [reply]
It wasn't planned, as it was not clear how much benefit it would bring. Note for example that in scikit-learn Varimax can be applied to Factor Analysis but not to PCA. If you are interested you can open a new issue to discuss this functionality, and how to implement it in a general way, so that future different rotations could be added later.
